### ISSUE TITLE
[fr] Update french translation

### ISSUE THREAD

**Issue by fego** [Zettlr/zettlr-docs#69]
The french translation is a bit old. Let's share here the files/folders we are working on, in order to avoid to work on the same files.
I am starting on the first file "citations.md" and I will continue with the other files of the "academic" folder.

**Comment by canpolat** [comment]
I'm curious as to how you (or other contributors) work on these translations. Do you work on the Git history to see that has been changed, or do you read through whole documents and compare the translation with the English version to see if it is still up-to-date?

**Comment by fego** [comment]
Because all files seem to need to be update I open them with an editor, the two files visible (I use the "split vertically" option in Intellij for example). If I'll manage to be up to date, I will look at the history, indeed.

**Comment by canpolat** [comment]
I guess you are right about the French translation. There has been many changes since the last update and going through whole files is a better approach.

I opened a new issue outlining an approach to keep translated files up-to-date. Hopefully, once you are done, my proposal will be also useful to you.

**Comment by nathanlesage** [comment]
I just pinned that issue, let's see where we can go from here!

**Comment by raphraph** [comment]
Is there still some work to do on the french translation ? I admit I often look up English docs because they tend to be more up to date and complete, and I didn't take the time to check if they are major differences for Zettlr between en and fr docs.

**Comment by nathanlesage** [comment]
@raphraph Yes, absolutely! Please feel free to overhaul anything you want in the French docs (and the English as well, if you spot some mistake!)
